### ISSUE TITLE
long token texts appears as duplicate

### ISSUE THREAD

**Issue by nrazon** [splitwise/TokenAutoComplete#383]
Hello @mgod 

There is view bug which I can't find a way to fix it :(
I downloaded you sample and It can be replicated on your github sample.

Problem is; If token text is longer than a line, even you elipsize toekns or not; token itself seems double. (but single clickable token, just view is double)

 To test it just change the mail address with a longer address like screenshot 2 at your sample Person.java file. 

Any idea how to fix it? 

![bug1](https://user-images.githubusercontent.com/662106/56732005-b1452500-6764-11e9-9bb4-e968a2625d31.jpg)
![bug2](https://user-images.githubusercontent.com/662106/56732006-b1452500-6764-11e9-924c-f5b910d280ac.PNG)

**Comment by nrazon** [comment]
@mgod Any idea how to fix this?

**Comment by mgod** [comment]
I'm not able to reproduce this in the current example app. Are you seeing this with the latest code? This could be Android version/keyboard/physical device specific.

**Comment by nrazon** [comment]
Yes it is latest code. (current example, and current latest library).
Using samsung galaxy note 8 (android 9).

But I tested now with samsung galaxy s7 (android 8) that is fine there. 
Maybe that is phone specific as you say. I will dig it more. But seems it is not general wide issue.  So I am closing. I will open again if I found any fix or more detaill.

Thx

> I'm not able to reproduce this in the current example app. Are you seeing this with the latest code? This could be Android version/keyboard/physical device specific.
